tests/recipe_tests/test_HDFReferenceRecipe.py

```diff
@@ -73,3 +73,44 @@ def test_multi(netcdf_local_file_pattern_sequential, tmpdir, with_intake):
         )
         ds = xr.open_dataset(m, engine="zarr", chunks={}, consolidated=False)
     assert (ds.foo == expected.foo).all()
+
+
+def _drop_lon(ds):
+    return ds.drop_vars(["lon"])
+
+
+@pytest.mark.parametrize("process", [None, _drop_lon])
+@pytest.mark.parametrize("with_intake", [True, False])
+def test_process(netcdf_local_file_pattern_sequential, tmpdir, with_intake, process):
+    file_pattern = netcdf_local_file_pattern_sequential
+    paths = [f for _, f in list(file_pattern.items())]
+    expected = xr.open_mfdataset(paths, engine="h5netcdf")
+    if process:
+        expected = _drop_lon(expected)
```

Comment:
```suggestion
        expected = process(expected)
```
This function is assigned to the name process in line 82, so you should be able to call it by this name.